docs/avt1guiance.md

```diff
@@ -0,0 +1,53 @@
+# Automated Verification Testing (AVT) stage 1
+
+## Introduction👋
+
+AVT1 testing is done with a browser extension developed by IBM called the Dynamic Assessment Plugin (DAP). It's a great first step to locking down and ensuring inclusivity and accessibility in your websites and components.
+
+## Installing DAP🔧
+
+1. Grab the plugin while connected to IBM's W3 [from here](https:/aat.w3ibm.mybluemix.net/auth/tools/dap.html).
```

Comment:
Can do shorten this link, too?

Reply:
I made a note in the PR that for some reason this link will not shorten. Snip marks it as 'invalid'. I'm open to other suggestions or tools though.